bindings/py/dummy.c

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+
+int main()
+{
+    return 0;
+}
+
+void initdummy() {}
```

Comment:
@rhyolight Yes. That works with swig building and allows for completion of `python setup.py sdist bdist bdist_wheel` on Windows (along with the `nupic.dummy` change below).